crates/re_renderer/src/view_builder.rs

```diff
@@ -124,6 +124,73 @@ pub enum Projection {
     },
 }
 
+impl Projection {
+    fn projection_from_view(self, aspect_ratio: f32) -> glam::Mat4 {
```

Comment:
I was toying with the idea of directly changing the projection matrix to be non-symmetrical, and when looking at the code I just _had_ to do this refactor. It's a pure refactor though, so don't worry.